src/model/formats/test-data.ts

```diff
@@ -178,12 +178,33 @@ export const EXPECTED_FOREFLIGHT_CONTENTS = {
               centered: false,
             },
             {
-              prompt: 'scary!',
+              prompt: 'scary multiline note!',
               expectation: '',
               type: ChecklistItem_Type.ITEM_WARNING,
               indent: 1,
               centered: false,
             },
+            {
```

Comment:
Do you need to update other format test files for these? (or is that what you're asking me to do?)